src/features/classification/security-controls/SecurityControls.js

```diff
@@ -66,7 +66,7 @@ class SecurityControls extends React.Component<Props, State> {
             items = getFullSecurityControlsMessages(controls, maxAppCount);
         } else {
             const shortMessage = getShortSecurityControlsMessage(controls);
-            items = shortMessage ? [shortMessage] : [];
+            items = shortMessage.message ? [shortMessage] : [];
```

Comment:
I think this should still return `null` when there's no message in order to simplify validating empty messages. I think the `message` property would ideally be required since there wouldn't be a `MessageItem` object without it